public/source/main.js

```diff
@@ -72,12 +72,16 @@ ko.bindingHandlers.autocomplete = {
         event.preventDefault();
         navigation.browseTo('repository?path=' + encodeURIComponent(value));
       } else if (localStorage.repositories && value.indexOf("/") === -1 && value.indexOf("\\") === -1) {
-        var folderNames = localStorage.repositories.replace(/("|\[|\])/g, "")
-          .split(",")
+        var folderNames = JSON.parse(localStorage.repositories)
           .map(function(value) {
+            var index = value.lastIndexOf("/");
+            if (index == -1) {
+              index = value.lastIndexOf("\\");
```

Comment:
You could use `ungit.config.fileSeparator` instead of these if statements

Reply:
Fixed.